data/blog/wolvctf-2023/wannaflag.mdx

```diff
@@ -0,0 +1,335 @@
+> From outside intelligence, we know the group sometimes goes by `w4nn4_fl4g`
+
+We’re now given a keyphrase to work with: `w4nn4_fl4g`. We can search for the specific term on Google by wrapping it in quotes, and our first result is a subreddit, [r/w4nn4_fl4g](https://www.reddit.com/r/w4nn4_fl4g/):
+
+![Reddit](/static/images/wolvctf-2023/reddit.png)
+
+Looking through the small amount of posts on this locked subreddit, we can find three users in particular which have access to post permissions (alongside one moderator): `u/w4nn4fl4g_admin`, `u/RemarkableDiamond443`, and `u/[deleted]`. None of the posts and memes were particularly interesting or relevant, but one thing stuck out in particular: the deleted user. We can find the username of the user through querying [camas.unddit.com](https://camas.unddit.com/) for a specific comment in `r/w4nn4_fl4g`; let’s utilize the comment “sorry” they left under [this post](https://www.reddit.com/r/w4nn4_fl4g/comments/11p6utf/questions/):
+
+![First Query](/static/images/wolvctf-2023/query1.png)
+
+We found the original username of the deleted user: `u/Chemical_Bread1558`! We can now query for posts the user made under the subreddit:
+
+![Second Query](/static/images/wolvctf-2023/query2.png)
+
+We’ve got a hit on a [moderator-deleted post](https://reddit.com/11p5w72). Let’s check it out:
+
+![Deleted Post](/static/images/wolvctf-2023/deleted-post.png)
+
+Well, that doesn’t really help. However, we can see the post’s original content by using the Unddit tool once again—simply replace the `reddit` in the URL with `unddit` to see deleted comments:
+
+![Unddit](/static/images/wolvctf-2023/unddit.png)
+
+The secret [website](wanna-flag-d60bf7cd-012a-4fcc-9a4c-e60eca6b653f-tlejfksioa-ul.a.run.app) provided leads us to this:
+
+![Website](/static/images/wolvctf-2023/website.png)
+
+We’ve managed to recover the flag from the first OSINT challenge, but it’s actually meant to be submitted as part of the third. The flag is `wctf{sp1nnnNn_tH3_cUb333e3E}`!
+
+## WannaFlag IV: Exfiltration
+
+> Now that we’ve successfully gotten into their website - I say we figure out what other data they have.  
+> Find and crack the master flag list, and submit the flag you see of ours on the list.
+
+**First blood!** We’re now tasked with analyzing the contents of their website. Scouring through the source code, we find two extra pages: `/ctfs.html` and `/prices.html`:
+
+![CTFs](/static/images/wolvctf-2023/website-ctfs.png)
+
+![Prices](/static/images/wolvctf-2023/website-prices.png)
+
+We’ve also got a footer with various social medias (which are all unsurprisingly rickrolls):
+
+![Footer](/static/images/wolvctf-2023/website-footer.png)
+
+However, on the `/ctfs.html` page, upon further inspection of the source code we find a secret page, `/data.html`, linked under the Instagram icon:
+
+```html ctfs.html
```

Comment:
The current MDX parser doesn’t seem to support file name annotation. We can try to add that if you feel it’d be a good thing to have.

Reply:
This MDX parser *does* actually support file name annotation, but its delimiter is a colon beside the code language (e.g. `` ```html:foo.html ``. See [`data/blog/new-features-in-v1.mdx`](https://github.com/jktrn/sekai.team/blob/master/data/blog/new-features-in-v1.mdx?plain=1#L26) for example.

Reply:
> The current MDX parser doesn’t seem to support file name annotation. We can try to add that if you feel it’d be a good thing to have.

Resolved with [`a808838` (#35)](https://github.com/project-sekai-ctf/sekai.team/pull/35/commits/a80883874671889a3ac3973ab14b3574b1b1bb2b).